Gentoo does not have ncurses5 headers anymore, and this fails with 6
when using wattrset that it currently expects to be a macro.
--- a/g_src/curses.h
+++ b/g_src/curses.h
@@ -9,2 +9,5 @@
 #else
+# ifndef NCURSES_WATTR_MACROS
+#  define NCURSES_WATTR_MACROS 1
+# endif
 # include <ncursesw/curses.h>
--- a/g_src/renderer_curses.cpp
+++ b/g_src/renderer_curses.cpp
@@ -296,3 +296,3 @@
       // We prefer libncursesw, but we'll accept libncurses if we have to
-      handle = dlopen("libncursesw.so.5", RTLD_LAZY);
+      handle = dlopen("libncursesw.so.6", RTLD_LAZY);
       if (handle) goto opened;
@@ -304,3 +304,3 @@
       if (handle) goto opened;
-      handle = dlopen("libncurses.so.5", RTLD_LAZY);
+      handle = dlopen("libncurses.so.6", RTLD_LAZY);
       if (handle) goto opened;
